src/navigation_controller.js

```diff
@@ -61,14 +61,6 @@ export class NavigationController {
    * @protected
    */
   addShortcutHandlers() {
-    if (Blockly.FieldColour) {
```

Comment:
Why are these unnecessary?

Reply:
These fields already have code for handling arrow keys, and it all does the correct things. The reason we needed extra code was because we were using something _other than_ arrow keys (WASD).

Reply:
We may need something like this if/when we allow remapping to other key sets. I think trying to design for that right now would be premature. Since it would also require monkey-patching, I'd rather just delete this code.